docs/cluster-configuration.md

```diff
@@ -166,6 +166,9 @@ For additional information, see here: [KubeSecondaryDNS](https://github.com/kube
 
 **Default**: `false`
 
+In order to set KSD's NameServerIP, set it on HyperConverged CR spec.KubeSecondaryDNSNameServerIP field.
+Default: empty string. Value is a string representation of IPv4.
```

Comment:
i think the value for that field should be validated at it is a valid IPv4 address, and reject update/creation of HCO CR otherwise.
the user should get immediate feedback if the value is invalid.

Reply:
Done